src/components/NFTBook/SpecTableItemAccessMethod.vue

```diff
@@ -3,7 +3,7 @@
     <NFTBookSpecTableLabel
       :text="$t('nft_details_page_label_access_methods')"
     />
-    <div class="font-[600]">{{ label }}</div>
+    <div :class="['font-[600]', labelStyle]">{{ label }}</div>
```

Comment:
What I mean is that `<NFTBookSpecTableValue />` also replaces this

this component accepts `preset` type e.g. 'campaign'

Reply:
[♻️ Refactor with <NFTBookSpecTableValue/>](https://github.com/likecoin/liker-land/pull/1883/commits/72644913263bd43ce5f2c751d3894e163e9d2119)